resources/js/processes/modeler/print/parseBpmnDocumentation/index.js

```diff
@@ -0,0 +1,34 @@
+import getFullyQualifiedNodeType from './typeParser';
+import { getNodeName, hasNonEmptyName } from './nameParser';
+import { nodeDocumentation, nodeText } from './documentationParser';
+
+function documentableBpmnNodes(bpmnString) {
+  const bpmnDoc = new DOMParser().parseFromString(bpmnString, 'text/xml');
+  const nodesWithNonEmptyId = Array.from(bpmnDoc.querySelectorAll('*[id]:not([id=""])'));
```

Comment:
I see what you're doing here, and maybe I'm just weird, but I feel like breaking out all the filters into their own intermediate variables somehow makes this harder to wrap my head around, as oppose to chaining the `filter`s off the original `Array.from`.

Maybe a structure like this would be cool:
```js
Array.from(/* ... */)
  .filter(removeThings)
  .filter(removeMoreThings)
  .map(convertToLeObject);

```

Reply:
would certainly read better! thanks